tests/integration/test_api/test_config/test_bruteforce_blocking_system/test_bruteforce_blocking_system.py

```diff
@@ -50,9 +50,8 @@ def test_bruteforce_blocking_system(tags_to_apply, get_configuration, configure_
     max_login_attempts = get_configuration['configuration']['access']['max_login_attempts']
 
     # Provoke a block from an unknown IP ('max_login_attempts' attempts with incorrect credentials).
-    for _ in range(max_login_attempts):
-        with pytest.raises(Exception):
-            get_api_details(user='wrong', password='wrong')
+    with pytest.raises(Exception):
```

Comment:
Maybe is better to use a less generic exception.

```suggestion
    with pytest.raises(RunTimeError):
```